lib/util/cache/repository/init.ts

```diff
@@ -10,23 +10,23 @@ export async function initRepoCache(config: RenovateConfig): Promise<void> {
   resetCache();
 
   const { platform } = GlobalConfig.get();
```

Comment:
Looks like `platform` is no longer used in this file and its references can be removed?

Reply:
it is still used as part of the first `if` check.
https://github.com/renovatebot/renovate/blob/2d71049650dfee6a8ba65c13b627422db79830b1/lib/util/cache/repository/init.ts#L15-L17

and I'm reliant on this check in some inner logic as i need the `platform` api

Reply:
platform should never be undefined here

Reply:
so should this check be removed? its part of the existing code so does the permalink above

Reply:
internally im using the `not null assertion` (!)